CONTRIBUTING.md

```diff
@@ -107,8 +107,7 @@ So we ask you to ensure all these are met before you shoot out a PR.
    Fixes: https://github.com/gitcoinco/web/issues/87
    Refs: https://github.com/gitcoinco/web/issues/91
    ```
-5. Ensure your squash your commit to make our history neater and reviewing
-   easier.
+5. Squashing your commits to make our history neater is always welcomed.
```

Comment:
Should probably preface this with instructions on how to access the `Squash & Merge` feature. n00b may not know how to? Also should probably read `Squashing and Merging your...` instead of just `Squashing your...`